magicbyirineu/Persistence/Repository/RealmTypes.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class RealmTypes {
```

Comment:
```suggestion
class DatabaseTypesService {
```

Plural tbm é um antipattern para nome de Classe.

Reply:
Refatorado!